packages/client/mutations/UpsertTeamPromptResponseMutation.ts

```diff
@@ -51,41 +39,70 @@ const mutation = graphql`
   }
 `
 
-const UpsertTeamPromptResponseMutation: StandardMutation<TUpsertTeamPromptResponseMutation> = (
-  atmosphere,
-  variables,
-  {onError, onCompleted}
-) => {
+export const upsertTeamPromptResponseUpdater: SharedUpdater<
```

Comment:
-1 This updater is not needed. Works for me without it.
As a rule of thumb, if you're returning an updated object from a mutation, the relay store will merge it into the existing object. Only case where an updater is required is if you only add part of the modified object (like an element in an array) and you need to update the parent.

Reply:
very much true! I removed the updater, optimistic updater, and replaced it with an optimistic response.